iotdb-core/node-commons/src/assembly/resources/sbin/destroy-datanode.bat

```diff
@@ -158,26 +158,6 @@ for %%i in (%dn_sync_dir%) do (
     )
 )
 
-for /f  "eol=# tokens=2 delims==" %%i in ('findstr /i "^pipe_receiver_file_dir"
```

Comment:
This cannot be removed since users can configure this to other directories than "dn_system_dir".

Reply:
Good catch! fixed